composition_test.go

```diff
@@ -242,7 +242,7 @@ func TestComposeBatchWontFail(t *testing.T) {
 
 	msg := <-c.Execute(makeMessage(StrMap{"input": 1}))
 	expected := StrMap{}
```

Comment:
Random rant. The `expected` map is initialised everywhere. Why not just create one in Recieve and return it?